rapids-cmake/cpm/package_override.cmake

```diff
@@ -56,10 +56,13 @@ as the override.
 
 .. note::
 
-  .. versionadded:: v23.10.00
+  .. versionadded:: v24.04.00
 
     When the variable `CPM_<package_name>_SOURCE` exists, any override entries
-    for `package_name` will be ignored.
+    for `package_name` will be parsed but ignored.
+
+    For versions v24.02 and v23.10 ( inclusive ) the variable `CPM_<package_name>_SOURCE`
```

Comment:
This was the behavior all the way up until 24.11 as well no?

Reply:
Updated the correct range 23.10 <-> 25.02